Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "protobuf-build"
-version = "0.9.2"
+version = "0.9.3"
```

Comment:
Should we bump to 0.10 as it is actually a breaking change?

Reply:
Yeah, it should, good catch!